src/consts.h

```diff
@@ -35,6 +35,10 @@
 #define IMU_BMI160 8
 #define IMU_ICM20948 9
 #define IMU_ICM42688 10
+
+#define IMU_MPU6500_NODMP 102
+#define IMU_MPU6050_NODMP 106
```

Comment:
Idk if I like these being defined this way... I think I'd prefer it just following the incrementing ID order.

Reply:
Using IDs 13 & 14 now(skipping ahead a bit to account for #296 and #308)